### PR TITLE
fix: stop log incorrectly

### PR DESCRIPTION
**Pull request by alex-Arc**

### CODE REVIEW

File: apps/server/src/services/runtime-service/RuntimeService.ts [hunk 1]
```diff
@@ -525,8 +525,8 @@ class RuntimeService {
     }
     const didStop = this.eventTimer?.stop();
     if (didStop) {
-      const newState = state.timer.playback;
-      logger.info(LogOrigin.Playback, `Play Mode ${newState.toUpperCase()}`);
+      const newPlaybackState = runtimeState.getState().timer.playback;
+      logger.info(LogOrigin.Playback, `Play Mode ${newPlaybackState.toUpperCase()}`);
```
Comment (cpvalente) [inline apps/server/src/services/runtime-service/RuntimeService.ts:529]: this was copy-pasted so we need to make the same change on the other functions
also, how about simplifying like 

```suggestion
      const newState = runtimeState.getState();
      logger.info(LogOrigin.Playback, `Play Mode ${newState.timer.playback.toUpperCase()}`);
```

Comment (alex-Arc) [reply apps/server/src/services/runtime-service/RuntimeService.ts:529]: think it was the only one that did not get new state after the playback action

**Review by cpvalente — APPROVED** [review]

File: packages/utils/src/validate-action/validatePlayback.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { Playback } from 'ontime-types';
  */
 export function validatePlayback(currentPlayback: Playback) {
   return {
-    start: currentPlayback !== Playback.Stop,
+    start: currentPlayback !== Playback.Stop && currentPlayback !== Playback.Play,
```
Comment (alex-Arc) [inline packages/utils/src/validate-action/validatePlayback.ts:8]: @cpvalente are you OK with this?
I saw that it was possible to click the play button in the UI while something was playing and that would generate error messages in the Log, nothing was breaking but....

Comment (cpvalente) [reply packages/utils/src/validate-action/validatePlayback.ts:8]: yeah, I cant remember if there was a use case for still letting it through. The change sounds good

**Pull request merged**